contracts/Vault.sol

```diff
@@ -45,6 +45,8 @@ contract Vault is
     // Constants
     //
 
+    bytes32 public constant MANAGER_ROLE =
```

Comment:
"manager" is a very generic word. it doesn't explain what this role is about
how about "INVESTOR_ROLE" instead?

Reply:
fixed